### ISSUE TITLE
Theme showing lock icon under pop-up window

### ISSUE THREAD

**Issue by dolenec** [iantrich/restriction-card#86]
**Checklist:**

- [x ] I updated to the latest version available
- [x ] I cleared the cache of my browser

**Release with the issue:**
1.2.7

**Last working release (if known):**

**Browser and Operating System:**
Windows OS, Android phone

**Description of problem:**

**Javascript errors shown in the web inspector (if applicable):**

```

```

**Additional information:**

This error was started under HA 2023.5.x

**Comment by dolenec** [comment]
![lovelace-3](https://github.com/iantrich/restriction-card/assets/16633222/5387bf3e-9b3f-45a2-af9d-969d21570376)

**Comment by Dehumanizer77** [comment]
Yes, same here, started to happen with the latest HA release :-/

**Comment by Dehumanizer77** [comment]
@iantrich do you think you'll be able to look into it? Maybe it is related to https://github.com/iantrich/restriction-card/issues/84 ?

**Comment by dolenec** [comment]
Is @iantrich still able to maintane plugin or should we make new fork and fix problem on new fork?

**Comment by Dehumanizer77** [comment]
@dolenec Looks like @iantrich is not available, if you are able to fix it, please do create a fork and you can send a pull request, in case @iantrich will be around in the future.

**Comment by ildar170975** [comment]
This [PR](https://github.com/iantrich/restriction-card/pull/89) fixed one thing but broke the whole card.

**Comment by Dehumanizer77** [comment]
It works perfectly for me..,

**Comment by bram2202** [comment]
> This [PR](https://github.com/iantrich/restriction-card/pull/89) fixed one thing but broke the whole card.

Do you have an example or something?

**Comment by ildar170975** [comment]
![image](https://github.com/iantrich/restriction-card/assets/71872483/2a991d9a-9688-4a8c-8550-39dcf3fd2f33)

**Comment by Dehumanizer77** [comment]
hmmm... looks like I still have the problem in some places...
![Screenshot_20230622-223051_Home Assistant](https://github.com/iantrich/restriction-card/assets/1876451/639fddc7-4a86-43d0-9611-a4138745e1d6)

**Comment by Dehumanizer77** [comment]
When I have restriction cards stacked like this, the fix doesn't work.

![Screenshot_20230622-223400_Home Assistant](https://github.com/iantrich/restriction-card/assets/1876451/d3eb49f1-c0c7-421e-8ef2-09676ee17468)

**Comment by ildar170975** [comment]
@Dehumanizer77 
Are these conventional button cards?

**Comment by Dehumanizer77** [comment]
Actually, it's custom button cards: https://github.com/custom-cards/button-card

**Comment by Dehumanizer77** [comment]
@ildar170975 This works for me:
  - card:
          entity: input_boolean.alarm
        type: custom:restriction-card
        action: hold
        row: true
   
maybe you need to specify action?

**Comment by Dehumanizer77** [comment]
or actually as I am looking at it again, does the restriction card type even apply to the boolean input?

**Comment by ildar170975** [comment]
> Actually, it's custom button cards

[It has its own lock feature.](https://github.com/custom-cards/button-card#lock-object)

**Comment by ildar170975** [comment]
> This works for me:

As I already said - the new card only works for `row: true`.
It does not work with cards since no `z-index` is specified now for overlay.

**Comment by ildar170975** [comment]
> does the restriction card type even apply to the boolean input in your example?

No, because overlay (which is used to overlap a card)  is now located BELOW the card.

**Comment by Dehumanizer77** [comment]
> > Actually, it's custom button cards
> 
> [It has its own lock feature.](https://github.com/custom-cards/button-card#lock-object)

ah, so probably a similar fix needs to be done there... thanks

**Comment by ildar170975** [comment]
> a similar fix needs to be done

Only if you are using its own lock feature and not using restriction-card.

**Comment by ildar170975** [comment]
I have another proposal (untested) - can we place the overlay AFTER the card/row?
(w/o using z-index)

**Comment by Dehumanizer77** [comment]
I have tried to lower the z-index as @iantrich suggested, but it doesn't fix the issue, the locks are stil visible on top when something else is opened on top. I have tried 10, 1 and even -50 with the same result. 
@ildar170975 if you can send me the changes you are proposing, I can test it, I was able to create a working fork.

**Comment by Dehumanizer77** [comment]
BTW, lowering z-index did actually work for the custom button-card, interestingly...

**Comment by ildar170975** [comment]
> if you can send me the changes you are proposing, I can test it, I was able to create a working fork.

Thanks a lot, but I cannot just give "some changes", they must be tested step by step by me((((



> the locks are stil visible

I will try to play with it too & report my results.

**Comment by Dehumanizer77** [comment]
Alternatively I can help you to create a fork with which you could test it if you haveany specific problem.
You can then add the forked repo as custom repository i  HACS to be able to test it.

**Comment by ildar170975** [comment]
This would be a great help from you!
Currently I am rather busy with some family matters; I will try to contact you as I will be able to!!!
